src/Sulu/Component/Localization/Localization.php

```diff
@@ -57,41 +58,47 @@ public static function createFromString($locale, $format = self::UNDERSCORE)
      * The language of the localization.
      *
      * @var string
+     * @Groups({"frontend", "Default"})
```

Comment:
Should we not also add the `ExclusionStrategy` we normally use on all entities?

Reply:
I am afraid of doing that, because that might result in some kind of BC break.

Reply:
Ok for me.